### ISSUE TITLE
No way to relearn messages with short text part

### ISSUE THREAD

**Issue by moisseev** [vstakhov/rspamd#556]
Messages with short text part cannot be learned, for obvious reason.
~~~
HTTP error: 400, <test@example.com> contains less tokens than required for bayes classifier: 8 < 11
~~~
But such messages are still being classified on scanning and it seems a good thing. On the other hand, there is no way to relearn such messages.

**Comment by moisseev** [comment]
Fixed with 44ba38e31c3ee6c5773ee805084099d51f7f46f8

**Comment by moisseev** [comment]
I've got `BAYES_HAM(-2.66)[]` for a short email (Classified, but options field is empty).
